resources/css/components/_link.css

```diff
@@ -84,6 +84,14 @@ a.cta {
     width: auto;
 }
 
+a.cta.disabled {
+    background: transparent;
+    border: var(--border-2) dashed;
+    border-color: var(--cta-border-hover);
+    color: var(--cta-color-hover);
+    pointer-events: none;
+}
```

Comment:
Styling for the disabled state should be handled similar to the disabled buttons. See: [_button.css](https://github.com/accessibility-exchange/platform/blob/019fa76639be469203461544f9bfb099af3dcf81/resources/css/components/_button.css#L258-L265)